src/client/client.go

```diff
@@ -469,7 +464,29 @@ func NewOnUserMachine(prefix string, options ...Option) (*APIClient, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "could not get active context")
 	}
+	return newOnUserMachine(cfg, context, prefix, options...)
+}
 
+// NewEnterpriseClientOnUserMachine constructs a new APIClient using $HOME/.pachyderm/config
```

Comment:
This might be out of scope, but since we were talking about UX. I personally don't like it when applications write to my $HOME. I think there are standards for `.config` and `.local` but no one ever follows them.

Reply:
I agree, another thing to reconsider for the product in the future. In this case we're reusing the existing pachyderm config and just adding a new field for the enterprise context, this comment is copy-paste from `NewOnUserMachine`.